modules/nw-sriov-networknodepolicy-object.adoc

```diff
@@ -61,12 +61,7 @@ If you specify both `pfNames` and `rootDevices` at the same time, ensure that th
 
 <12> An array of one or more PCI bus addresses for the PF of the device. Provide the address in the following format: `0000:02:00.1`.
 
-<13> Optional: The platform specific filter string. The only supported platform is `openstack` and allowed value is `openstack/NetworkID:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`.
-+
-[NOTE]
-====
-`xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx` is the `network_id` from `/var/config/openstack/latest/network_data.json` metadata file.
-====
+<13> Optional: The platform specific filter string. The only supported platform is {rh-openstack-first}. Acceptable values are of the following format: `openstack/NetworkID:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`. Replace `xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx` with the value from the `/var/config/openstack/latest/network_data.json` metadata file.
```

Comment:
Nice change!

Reply:
In addition, the administrator should also know the OpenStack Network UUID apriori as part of network planning.  The OpenStack networks are created to glue together the different VNF/CNF components in the system and have been architected to carry the necessary information.